cmd/operator-sdk/build/cmd.go

```diff
@@ -17,7 +17,6 @@ package build
 import (
```

Comment:
Changes to `build` can't happen until #3433 is merged, which uses the legacy command.